public/PROPOSAL/math/Function.h

```diff
@@ -44,7 +44,7 @@ class Polynom {
     Polynom(const Polynom&);
 
     ~Polynom() {};
-    // Polynom& operator=(const Polynom& poly);
+    Polynom& operator=(const Polynom& poly) = default;
```

Comment:
https://stackoverflow.com/questions/51863588/warning-definition-of-implicit-copy-constructor-is-deprecated

Good explanation